aws/cloudformation/s3_backed_subdomain.yml

```diff
@@ -0,0 +1,112 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+Description:
+  "Basic template for provisioning subdomains which serve content from S3 via
```

Comment:
Can we add a note here explaining that this template is only used for provisioning subdomains on the production domain and that are used by all (including development) environments? Also note that changes committed to this template are not automatically applied to Stacks by our Continuous Integration / Deployment processes?